4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-general-editor-execute-editorscripts-using-the-command-palette.md

---
type: entry
section: general
subsection: editor
rank: 6
importance: 3
anchor: entry-general-editor-execute-editorscripts-using-the-command-palette
title: "Execute `EditorScript`s using the command palette"
text: |

>   Your named project `EditorScript` files now appear in the command palette, making it much easier to execute specific project commands.

Reviewer: ```suggestion
  Your named `EditorScript` files now appear in the command palette, making it much easier to execute specific project commands.
```
Or
```suggestion
  Any named `EditorScript` files in your project now appear in the command palette, making it much easier to execute specific project commands.
```

>   Your named project `EditorScript` files now appear in the command palette. It will be much easier to execute your specific project commands from now on.

Reviewer 2: ```suggestion
  Your named project `EditorScript` files now appear in the command palette, making it much easier to execute specific project commands.
```
